qml/translations/StoreDescription.qml

```diff
@@ -0,0 +1,22 @@
+import QtQuick 2.0
+
+Item {
+    // providing dummy translations for app descriptions shown on Store
+    function qsTrIdString() {
+        //% "Documents you download to the device appear and can be viewed in Documents app."
+        QT_TRID_NOOP("sailfish-office-la-store_app_summary")
+
+        //% "Documents lists every document stored on you device, SD card or other connected storage device. Documents can be viewed, zoomed, you can also jump to a specific part of the document from an attached page. In addition PDF viewer supports text search, annotations and adding comments.\n"
```

Comment:
"can be viewed, zoomed, you can also" is missing and somewhere.

Also the line is quite long, github doesn't really handle it any better.